app/views/layouts/application.html.erb

```diff
@@ -53,12 +53,16 @@
         </form> -->
         <ul class="nav navbar-nav navbar-right">
           <% if current_user %>
+            <% if current_user.subscribed %>
+              <li><%= link_to current_user.email, account_path %></li>
```

Comment:
I see what you mean about needing a separate page for "signed in but unsubscribed".  Disregard my previous comment and carry on as-is for now.